index.rb

```diff
@@ -126,16 +126,19 @@ def show(**kwargs)
   end
 end
 
-venues << Venue.new(:name => 'DNA Lounge', :link => 'https://www.dnalounge.com/calendar/dnalounge.rss') do
+venues << Venue.new(:name => 'DNA Lounge', :link => 'https://www.dnalounge.com') do
   # Regex to dig the calendar link out of the description
   link_regex = Regexp.new(/https:\/\/www.dnalounge.com\/calendar\/\d{4}\/[\d\w\-]+\.html/i)
 
-  URI.open(link) do |xml|
+  URI.open(URI.join(link, 'calendar/dnalounge.rss')) do |xml|
     Nokogiri::XML(xml).css('item').map do |item|
       description = item.css('description').text
 
+      time = item.css('dnalounge|time').text
+      date = item.css('dnalounge|date').text
+
       show(
-        time: Time.parse(item.css('dnalounge|date').text),
+        time: Time.parse("#{time} #{date}"),
```

Comment:
Maybe `<pubDate>` would be better here? It looks like it's a formatted timestamp for the show.

Reply:
Oh that's way better than what I'm doing. Fixed!